docs/getting-started/documentation/pull-data-transfer.md

```diff
@@ -34,11 +34,15 @@ When initiating the transfer, select "Custom Transfer Process JSON", and provide
 ```json
 {
   "@type": "https://w3id.org/edc/v0.0.1/ns/TransferRequest",
+  "https://w3id.org/edc/v0.0.1/ns/assetId": "{{ASSET_ID}}",
```

Comment:
documentation changes fixes are Patch changes for the CHANGELOG, we can add an entry for that and prefix it with "Docs: "